Assets/HoloToolkit/BuildAndDeploy/Editor/BuildDeployTools.cs

```diff
@@ -22,6 +22,26 @@ public class BuildDeployTools
     {
         public static readonly string DefaultMSBuildVersion = "14.0";
 
+        public static bool CanBuild()
+        {
+            if (PlayerSettings.GetScriptingBackend(BuildTargetGroup.WSA) == ScriptingImplementation.IL2CPP && Il2CppAvailable())
+            {
+                return true;
+            }
+
+            return PlayerSettings.GetScriptingBackend(BuildTargetGroup.WSA) == ScriptingImplementation.WinRTDotNET && DotNetAvailable();
+        }
+
+        public static bool DotNetAvailable()
```

Comment:
nit: I have a preference for naming these function `IsDotNetAvailable` and `IsIl2CppAvailable`

Reply:
Good point.  Sorry I usually do the same.